### DISCUSSION TITLE
[Esame 23 Gennaio 2019] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#222] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/c0fc3afb-245c-49c5-8d16-fb1101936b16)

**Comment by LeoCosta02** [comment]
uguale all'esercizio dato nel foglio 3 es1 
- #147
